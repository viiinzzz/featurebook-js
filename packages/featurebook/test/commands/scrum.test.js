/* eslint-disable camelcase */
/* eslint-disable quotes */
const {
  given, when, then_wait_at_most, expect,
  node,
  one_minute,
} = require('../test.helpers');

given('I have a folder with .feature files', () => {
  const folder = '../../test/features';
  when('I run command ‘scrum’', () => {
    then_wait_at_most(one_minute)
      .that('it generates expected output to the console', async () => {
        const str = await node({
          program: 'bin/featurebook',
          options: `scrum ${folder}`,
        });
        expect(str).toMatchSnapshot();
      });
  });
});
