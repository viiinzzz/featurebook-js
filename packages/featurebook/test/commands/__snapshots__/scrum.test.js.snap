// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`I have a folder with .feature files I run command ‘scrum’ it generates expected output to the console 1`] = `
"featurebook v3.0.0-b.6 scrum:


┌─────────────────────────────────────────────┐
│                 [1mundefined [22m                  │
│              [36mfeatures[39m[36m - brief[39m               │
├─────────────────────────────────────────────┤
│ Sprint      Feature title  Plane  Status    │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mAct 1 Scene 1[39m         I#11      │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mAct 1 Scene 2[39m         I#12      │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mAct 1 Scene 3[39m         I#13      │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mAct 1 Scene 4[39m         I#14      │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mAct 1 Scene 5[39m         I#15      │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
├─────────────────────────────────────────────┤
│ [35m› S1[39m        [32mhome.page[39m             I#314     │
│ [37m2022-10-31[39m                                  │
│ [37m2022-11-30[39m                        [34m!proposed[39m │
│                                             │
└─────────────────────────────────────────────┘

"
`;
